src/packages/src/xroad/ubuntu/generic/xroad-center.cron.d

```diff
@@ -5,3 +5,6 @@
 
 # backup retention policy, delete backups older that 30 days
 10 * * * * xroad find /var/lib/xroad/backup -type f -name "cs-automatic-backup*.gpg" -mtime 30 -delete
+
+# reload nginx, if configuration https certificates have changed
+find /etc/xroad/ssl -type f -regextype posix-extended -regex '.*(internal|external)-conf\.crt' -mmin -60 -exec sudo service nginx reload {} +
```

Comment:
Good catch, I somehow misted that missing cron expression and user. The find command itself had worked, but this time I wanted to test the cron itself and other install related things with the jenkins install-test job (I feel testing cron on my local docker has some limitations). Did a separate script and tweaked some more to make it work (had to also add a sudoers entry so that the scripts could reload nginx without password prompt.)